Dockerfile.interop

```diff
@@ -39,15 +39,15 @@ COPY interop_binaries /src/interop_binaries
 COPY messages /src/messages
 COPY tools /src/tools
 COPY xtask /src/xtask
-ARG BINARY
-RUN cargo build --features fpvec_bounded_l2 --profile $PROFILE -p janus_interop_binaries \
-    --bin $BINARY
+RUN cargo build --features fpvec_bounded_l2 --profile $PROFILE -p janus_interop_binaries
```

Comment:
nit: `janus_interop_binaries` is a bit of an awkward name for a binary, I'd just call it `janus_interop` if it's not too much work to plumb through.